src/main.c

```diff
@@ -1292,7 +1267,7 @@ int main(int argc, char *argv[])
 		}
 	}
 	// tranform possibly UTF-8 encoded custom layout string to UTF-16
-	str2wcs(customLayoutWcs, customLayout, 33);
+	MultiByteToWideChar(CP_UTF8, 0, customLayout, -1, customLayoutWcs, 33);
```

Comment:
This is a regression. If you want to try a custom layout (for example swapping ü and ö) and pass it as command line argument in `cmd.exe`:

```
neo-llkh.exe customLayout="qwertzuiopöasdfghjklüäyxcvbnm,.-"
```

With your change, this doesn't work anymore. The keys for ä, ö, and ü print white boxes or boxes with question marks depending on the program.

Reply:
dropped the commit